src/algebra/gcd_monoid.lean

```diff
@@ -3,9 +3,9 @@ Copyright (c) 2018 Johannes Hölzl. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Johannes Hölzl, Jens Wagemaker
 
-GCD domain and integral domains with normalization functions
+GCD monoid and integral monoids with normalization functions
```

Comment:
Does this still make sense?
Maybe you can write a module docstring for this file now?